GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -161,116 +151,12 @@ public class Glia {
-        }
-
-        let viewFactory = ViewFactory(
-            with: theme,
-            messageRenderer: messageRenderer,
-            environment: .init(
-                data: environment.data,
-                uuid: environment.uuid,
-                gcd: environment.gcd,
-                imageViewCache: environment.imageViewCache,
-                timerProviding: environment.timerProviding,
-                uiApplication: environment.uiApplication
-            )
-        )
-        startRootCoordinator(
-            with: interactor,
-            viewFactory: viewFactory,
-            sceneProvider: sceneProvider,
-            engagementKind: engagementKind,
-            features: features
-        )
-    }
-
-    /// Starts the engagement.
-    ///
-    /// - Parameters:
-    ///   - engagementKind: Engagement media type.
-    ///   - configuration: Engagement configuration.
-    ///   - queueID: Queue identifier.
-    ///   - visitorContext: Visitor context.
-    ///   - theme: A custom theme to use with the engagement.
-    ///   - sceneProvider: Used to provide `UIWindowScene` to the framework. Defaults to the first active foreground scene.
-    ///
-    /// - throws:
-    ///   - `SalemoveSDK.ConfigurationError.invalidSite`
-    ///   - `SalemoveSDK.ConfigurationError.invalidEnvironment`
-    ///   - `SalemoveSDK.ConfigurationError.invalidRegionEndpoint`
-    ///   - `SalemoveSDK.ConfigurationError.invalidSiteApiKey`
-    ///   - `SalemoveSDK.ConfigurationError.invalidAppToken`
-    ///   - `GliaError.engagementExists`
-    ///
-    public func start(
-        _ engagementKind: EngagementKind,
-        configuration: Configuration,
-        queueID: String,
-        visitorContext: VisitorContext?,
-        theme: Theme = Theme(),
-        features: Features = .all,
-        sceneProvider: SceneProvider? = nil
-    ) throws {
-        try configure(
-            with: configuration,
-            queueId: queueID,
-            visitorContext: visitorContext
-        ) { [weak self] in
-
-            self?.setChatMessageRenderer(messageRenderer: .webRenderer)
-
-            do {
-                try self?.startEngagement(
-                    engagementKind: engagementKind,
-                    theme: theme,
-                    features: features,
-                    sceneProvider: sceneProvider
-                )
-            } catch {
-                print("Engagement has not been started. Error='\(error)'.")
-            }
-        }
-    }
-
     /// Maximizes engagement view if ongoing engagment exists.
     /// Throws error if ongoing engagement not exist.
     /// Use this function for resuming engagement view If bubble is hidden programmatically and you need to
     /// present engagement view.
     public func resume() throws {
-        guard engagement != .none else {
-            throw GliaError.engagementNotExist
-        }
+        guard engagement != .none else { throw GliaError.engagementNotExist }
```

Comment:
I never saw this when it was created. Unfortunately it's too late, because gramatically it doesn't sound good. `engagementInexistent` or `engagementDoesNotExist` would have been better.

Reply:
yep, it has been introduced a long time ago...